activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryProcessInstancesEntityIT.java

```diff
@@ -261,6 +263,52 @@ public void shouldGetAdminProcessInfo() {
             });
     }
 
+    @Test
+    public void shouldDeleteProcessInfo() {
+        //given
+        ProcessInstance process = processInstanceBuilder.startSimpleProcessInstance("simple");
+
+        eventsAggregator.sendAll();
+
+        await()
+            .untilAsserted(() -> {
+                identityTokenProducer.withTestUser("hradmin");
+
+                ResponseEntity<ProcessInstance> responseEntity = testRestTemplate.exchange(
+                    ADMIN_PROC_URL + "/" + process.getId(),
+                    HttpMethod.GET,
+                    identityTokenProducer.entityWithAuthorizationHeader(),
+                    new ParameterizedTypeReference<>() {}
+                );
+                //then
+                assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
+                assertThat(responseEntity.getBody()).isNotNull();
+                assertThat(responseEntity.getBody().getId()).isNotNull();
+
+                ProcessInstance responseProcess = responseEntity.getBody();
+                assertThat(responseProcess.getId()).isEqualTo(process.getId());
+
+                eventsAggregator.addEvents(new CloudProcessCancelledEventImpl(responseProcess));
+                eventsAggregator.addEvents(new CloudProcessDeletedEventImpl(responseProcess));
+            });
+
+        eventsAggregator.sendAll();
+
+        await()
+            .untilAsserted(() -> {
+                identityTokenProducer.withTestUser("hradmin");
+
+                ResponseEntity<ProcessInstance> responseEntity = testRestTemplate.exchange(
+                    ADMIN_PROC_URL + "/" + process.getId(),
+                    HttpMethod.GET,
+                    identityTokenProducer.entityWithAuthorizationHeader(),
+                    new ParameterizedTypeReference<>() {}
+                );
+                //then
+                assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.NOT_FOUND);
```

Comment:
Shouldn't we add coverage for related entities as well (tasks, candidate groups, etc)?